src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.*; 
+
+/**
+ * Represents tagging in the address book.
+ * Guarantees: immutable
+ */
+public class Tagging {
+	
+	private char _addOrRemoveSign;
```

Comment:
Indentation problem here. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.